modules/core/src/core/animation-loop.js

```diff
@@ -10,7 +10,7 @@ import {getPageLoadPromise} from '../webgl/context';
 import {isWebGL, requestAnimationFrame, cancelAnimationFrame} from '../webgl/utils';
 import {log} from '../utils';
 import assert from '../utils/assert';
-import {Stats} from 'probe.gl';
+import luma from '../init';
```

Comment:
If we import it like this, whenever we use it, we don't need to add it to `.eslintrc`?

Reply:
The issue I had was in the examples, since the `luma` object isn't exported from Luma.gl. Should we export it?

Reply:
> Should we export it?

In that case we should probably export the `stats` object, not the luma wrapper object.

Reply:
Sounds good.